crates/uv-cli/src/lib.rs

```diff
@@ -4394,9 +4395,9 @@ pub struct PublishArgs {
     #[arg(default_value = "dist/*")]
     pub files: Vec<String>,
 
-    /// The URL of the upload endpoint.
+    /// The URL of the upload endpoint (not the simple index URL!).
     ///
-    /// Note that this typically differs from the index URL.
+    /// Note that there are typically different URLs for index access ("simple") and index upload.
```

Comment:
Should we say something like this?

I don't think users struggling with this will now what the simple index API is?

> Note that there are typically different URLs for index access (e.g., `.../simple`) and index upload.